transactionoutbox-core/src/test/java/com/gruelbox/transactionoutbox/acceptance/AbstractAcceptanceTest.java

```diff
@@ -56,7 +74,7 @@ abstract class AbstractAcceptanceTest {
    * Uses a simple direct transaction manager and connection manager and attempts to fire an
    * interface using a custom instantiator.
    */
-  @Test
+  // @Test
```

Comment:
Tests have been disabled?

Reply:
ooops...did that in local because the test execution lasts forever